elasticsearch_collectd.py

```diff
@@ -192,6 +192,21 @@
     'process.cpu.percent': Stat("gauge", "nodes.%s.process.cpu.percent"),
 }
 
+NODE_STATS_ES_2 = {
+    'indices.cache.filter.evictions':
+            Stat("counter", "nodes.%s.indices.query_cache.evictions"),
+    'indices.cache.filter.size':
+        Stat("gauge", "nodes.%s.indices.query_cache.cache_size"),
+    'indices.cache.filter.hit-count':
```

Comment:
Sorry about that! Here is the fix https://github.com/signalfx/collectd-elasticsearch/pull/20/files